The name 'GetFile' does not exist in the current context

I have tried to add the following code to a task:
```cs
Squirrel(GetFile(path));
```
It didn't work and I kept having the following error:

> Error occurred when compiling build script: E:/Dev/build.cake(73,18): error CS0103: The name 'GetFile' does not exist in the current context

I have managed to make it work with this code though:
```cs
Squirrel(new FilePath(path));
```

Have I done something wrong or is the documentation out of date?

It is out of date.  That is for the issue and I will get it all updated.

On Tue, Oct 17, 2017, 2:50 PM Damien Chaib <notifications@github.com> wrote:

> I have tried to add the following code to a task:
>
> Squirrel(GetFile(path));
>
> It didn't work and I kept having the following error:
>
> Error occurred when compiling build script: E:/Dev/build.cake(73,18):
> error CS0103: The name 'GetFile' does not exist in the current context
>
> I have managed to make it work with this code though:
>
> Squirrel(new FilePath(path));
>
> Have I done something wrong or is the documentation out of date?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cake-contrib/Cake.Squirrel/issues/22>, or mute the
> thread
> <https://github.com/notifications/unsubscribe-auth/ABfthZM8aGHap4eZWGverMnAxpZQ0eAyks5stPbhgaJpZM4P8pWT>
> .
>

--- a/src/Cake.Squirrel/SquirrelAliases.cs
+++ b/src/Cake.Squirrel/SquirrelAliases.cs
@@ -20,7 +20,7 @@ namespace Cake.Squirrel {
         /// 
         /// Task("PackageNoSettings")
         ///  .Does(() => {
-        ///    Squirrel(GetFile("Package.nupkg"));
+        ///    Squirrel(File("Package.nupkg"));
         /// });
         /// </code>
         /// </example>
@@ -46,7 +46,7 @@ namespace Cake.Squirrel {
         ///    settings.NoMsi = true;
         ///    settings.Silent = true;
         /// 
-        ///    Squirrel(GetFile("Package.nupkg"), settings);
+        ///    Squirrel(File("Package.nupkg"), settings);
         /// });
         /// </code>
         /// </example>
@@ -78,7 +78,7 @@ namespace Cake.Squirrel {
         ///    settings.NoMsi = true;
         ///    settings.Silent = true; 
         /// 
-        ///    Squirrel(GetFile("Package.nupkg"), settings, true, false);
+        ///    Squirrel(File("Package.nupkg"), settings, true, false);
         /// });
         /// </code>
         /// </example>

